code/game/gamemodes/blob/blobs/core.dm

```diff
@@ -63,7 +63,8 @@
 
 /obj/effect/blob/core/Life()
 	if(!overmind)
-		create_overmind()
+		if(!qdeleted(src))
```

Comment:
The only times I've seen it happen is when a blob core is killed and deletes its overmind sooooo